### ISSUE TITLE
Daddy live sports schedule broken

### ISSUE THREAD

**Issue by karlo2105** [mrgsi/mrgsi.github.io#14]
Daddylive sports is broken. It would be great to add custom url settings instead of preselected addresses because none of them work.

**Comment by mrgsi** [comment]
Live channels still works so it's not the URL, it's the actual website (all versions) that has made a change to the Live Sports section. That means someone will need to re-code it. Unfortunately, that's above my pay grade but I've got an eye out for any new versions.

**Comment by karlo2105** [comment]
They only changed url. Those are currently working addresses : 
https://dlhd.sx/
https://daddylive.watch/home.php
https://www.daddylivehd.one/

Could you just replace preselected addresses by empty column url settings where we could fill in working urls in order to get back live sports section ?

**Comment by Better-Red** [comment]
Hello, there is an updated version of Daddy Live with a working sports section in The Crew repo. Any chance of having a look at their changes?

**Comment by mrgsi** [comment]
It's been changed in the repository but you will need to manually force update it. The versioning got a little wonky as it was passed through several hands. The Crew updated the original version (from a dev who didn't want to maintain it) when it broke the first time. Then dEXE took it, updated it to 3.0, then slapped his name on it. Then I added a code to fix a Kodi 21 issue (provided by someone else). That took it to version 3.0.1. However, this new version The Crew went back to their own versioning series and took it from 2.5 to **version 2.8**. There is also another version by dEXE 4.0.0 which is broken and has some bad graphics. 
As long as The Crew (the original forker) maintain it, I will defer to their versions avoiding future confusion of a daddylive multiverse. Since it's a lower version number than what I had previously, Kodi won't see it as an update.
Long story short, it's the "fixed version" now. lol

**Comment by karlo2105** [comment]
@mrgsi
Could you please add possibility to select channels in livesports section because we don't know what channels air specific sports and we are limited to first choice only ?

**Comment by Better-Red** [comment]
> It's been changed in the repository but you will need to manually force update it. The versioning got a little wonky as it was passed through several hands. The Crew updated the original version (from a dev who didn't want to maintain it) when it broke the first time. Then dEXE took it, updated it to 3.0, then slapped his name on it. Then I added a code to fix a Kodi 21 issue (provided by someone else). That took it to version 3.0.1. However, this new version The Crew went back to their own versioning series and took it from 2.5 to **version 2.8**. There is also another version by dEXE 4.0.0 which is broken and has some bad graphics. As long as The Crew (the original forker) maintain it, I will defer to their versions avoiding future confusion of a daddylive multiverse. Since it's a lower version number than what I had previously, Kodi won't see it as an update. Long story short, it's the "fixed version" now. lol

Thanks @mrgsi

**Comment by mrgsi** [comment]
Updated to 2.9
